zuul.d/layout.yaml

```diff
@@ -11,25 +11,27 @@
       tox_envlist: py38
     timeout: 3600
 
-- job:
-    name: molecule-vagrant-centos
-    description: Run py36 tox environment
-    parent: ansible-tox-py36
-    nodeset: centos-8-1vcpu
-    attempts: 2
-    vars:
-      tox_envlist: py36
-    timeout: 3600
+# CentOS is unsupported due to:
+# https://github.com/vagrant-libvirt/vagrant-libvirt/issues/1127
```

Comment:
maybe it worth mentioning https://github.com/hashicorp/vagrant/issues/11020 too ?